cmd/kubectl-direct_csi/install.go

```diff
@@ -90,24 +88,15 @@ func install(ctx context.Context, args []string) (err error) {
 	if err != nil {
 		return fmt.Errorf("invalid tolerations. format of '--tolerations' must be <key>[=value]:<NoSchedule|PreferNoSchedule|NoExecute>")
 	}
-	defaultAuditDir, err := utils.GetDefaultAuditDir()
-	if err != nil {
-		return fmt.Errorf("unable to get default audit directory; %w", err)
-	}
-	if err := os.MkdirAll(defaultAuditDir, 0700); err != nil {
-		return err
-	}
 
-	file, err := utils.NewSafeFile(fmt.Sprintf("%v/%v-%v", defaultAuditDir, auditInstall, time.Now().UnixNano()))
+	file, err := utils.WritetoAuditFile(auditInstall)
 	if err != nil {
-		return fmt.Errorf("unable to get default audit directory ; %w", err)
+		return err
```

Comment:
You don't want to ignore the error this time?

Reply:
logged the error